Dockerfile

```diff
@@ -73,10 +73,31 @@ RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=$SCCACHE_DIR,sharing=locked \
     cargo build --release --features="$FEATURES" --package=${RBUILDER_BIN}
 
+#
+# Reproducible builder container (deterministic source-date-epoch, no caching, no incremental builds)
+#
+FROM base AS rbuilder-reproducible
+ARG RBUILDER_BIN
+ARG FEATURES
+WORKDIR /app
+COPY . .
+RUN SOURCE_DATE_EPOCH=$(git log -1 --pretty=%ct) \
```

Comment:
This will force docker to build this 2 times (and one of them would be slow build), let's do it better

Reply:
All set :+1: